packages/jest-runtime/src/index.ts

```diff
@@ -2129,8 +2129,8 @@ export default class Runtime {
       mock,
       mocked,
       now: () => _getFakeTimers().now(),
-      requireActual: this.requireActual.bind(this, from),
-      requireMock: this.requireMock.bind(this, from),
+      requireActual: moduleName => this.requireActual(from, moduleName),
+      requireMock: moduleName => this.requireMock(from, moduleName),
```

Comment:
Is this fine? Not sure I understand what `bind` does here. Rework it, because I couldn’t find a clean way to return `T` through `bind`. Only work around was though casting, but this way it looked cleaner. All test are passing locally, let’s see what CI thinks.